src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -1199,9 +1196,7 @@ public void readFully(byte[] buf) throws IOException {
      */
     public void readFully(byte[] buf, int off, int len) throws IOException {
         int endoff = off + len;
-        if (off < 0 || len < 0 || endoff > buf.length || endoff < 0) {
-            throw new IndexOutOfBoundsException();
-        }
+        Objects.checkFromToIndex(off, endoff, buf.length);
```

Comment:
Why not using `Objects.checkFromIndexSize(off, len, buf.length);` just like in the previous change at line 1029 above?

Reply:
Good point, done!